kong/plugins/hmac-auth/api.lua

```diff
@@ -53,5 +53,34 @@ return{
     DELETE = function(self, dao_factory)
       crud.delete(self.hmacauth_credential, dao_factory.hmacauth_credentials)
     end
+  },
+  ["/hmac-auths/"] = {
+    GET = function(self, dao_factory)
+      crud.paginated_set(self, dao_factory.hmacauth_credentials)
+    end
+  },
+  ["/hmac-auths/:credential_username_or_id/consumer"] = {
```

Comment:
Here, wouldn't `hmac_username` be a better name as well?

Reply:
Just to make sure `/hmac-auths/:hmac_username_or_id/consumer`, right?

Reply:
I think that would be slightly better yes. "credential" is ok for key-auth or basic-auth I would say, but here I think we need users to grasp better where is this "username" attribute we are referring to. It would thus be the username of the `/hmac-auth` entities

Reply:
@thibaultcha Should I update Line 23 in this file as well to maintain consistency? Also, I'll update tests.